binary/util.py

```diff
@@ -236,7 +236,8 @@ def __init__(self, encoder):
             'List_Data': partial(self.encode_multi_frame_list, encoder=self.encode_data_frame),
             'List_ScheduledTaskHandler': partial(self.encode_multi_frame_list, encoder=self.encoder.custom_type_encoder
                                                  .encoder_for('ScheduledTaskHandler')),
-            'SqlPage': partial(self.encode_sqlpage)
+            'SqlPage': partial(self.encode_sqlpage),
+            'HazelcastJsonValue': partial(self.encode_json)
```

Comment:
I guess we won't need those anymore

Reply:
I believe we need this for proper usage of ```HazelcastJsonValue```. That type was introduced in ```2.4```, but nobody uses it. When I tried to use it the generated tests failed.